check_plugins_viewer_mandatory_fields.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+FIELDS=("title" "publisher" "category" "icon" "description" "repository" "firstPublicationDate" "latestUpdateDate")
+CATEGORIES=("Editor" "Debugger" "Formatter" "Language" "Linter" "Snippet" "Theme" "Other")
+
+function check_category() {
+for CATEGORY in "${CATEGORIES[@]}"
+do
+  if [[ ${CATEGORY} == "$1" ]];then
+    return 0
+  fi
+done
+return 1
+}
+
+cd plugins || exit 1
+for d in */ ; do
+  ID_DIR_NAME=${d%/}
+  cd "$d" || exit 1
+
+  for VERSION_DIR_NAME in */ ; do
+    # Remove trailing slash
+    VERSION_DIR_NAME=${VERSION_DIR_NAME%/}
+    cd "${VERSION_DIR_NAME}" || exit 1
+
+    unset NULL_OR_EMPTY_FIELDS
+
+    for FIELD in "${FIELDS[@]}"
```

Comment:
Can you elaborate on whether we check unknown fields here and whether we should do that?

Reply:
no, we don't check for them here